src/googleMusic.ts

```diff
@@ -71,25 +99,29 @@ export default class GoogleMusic {
                 )
               })
           } else {
-            codeCache.put('authCode', gMusicResponse.payload)
+            codeCache.put('authCode', gMusicChannel.payload)
           }
           break
         case 'playState':
-          this._playState = gMusicResponse.payload
+          this._playState = gMusicChannel.payload
           break
         case 'track':
-          this._track = gMusicResponse.payload
+          this._track = gMusicChannel.payload
           this.refreshStatusBar()
           break
         case 'rating':
-          this._rating = gMusicResponse.payload
+          this._rating = gMusicChannel.payload
           break
         case 'shuffle':
-          this._shuffle = gMusicResponse.payload
+          this._shuffle = gMusicChannel.payload
           break
         case 'repeat':
-          this._repeat = gMusicResponse.payload
+          this._repeat = gMusicChannel.payload
+          break
+        case 'time':
           break
+        default:
+          console.log(gMusicChannel.channel)
```

Comment:
There's not a lot of reason to log unsupported channels without context... it adds unnecessary clutter to the console. Let's take it out.